### DISCUSSION TITLE
Custom tsconfig paths aren't recognized

### DISCUSSION THREAD

**Discussion by gopeter** [zed-industries/zed#6603] [Feature Requests]
This works in VSCode but not in Zed:

```
{
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "@app/*": [
        "app/packs/app/*"
      ],
      "@shared/*": [
        "app/packs/shared/*"
      ],
      "*": [
        "app/packs/*"
      ]
    }
}
```

<img width="1183" alt="image" src="https://github.com/zed-industries/community/assets/1242960/7e959050-ff8f-4533-a3db-85236e9842f7">

Not sure what I could do to make this work 🤔

**Comment by yagobatista** [comment]
Same problem here. Did you find any solutions?

**Reply by flaviomreis** [reply]
Same here in Zed 0.147.2

**Comment by robertwt7** [comment]
<img width="582" alt="Screenshot 2024-10-30 at 12 41 42 PM" src="https://github.com/user-attachments/assets/46bb10f0-46ec-4d82-84dd-bdbd97bfe333">

I can't even get zed to recognise my tsconfig on my root project. how did you guys set this up?

**Comment by mikayla-maki** [comment]
Hi y'all, I made a minimal repro of this in a test project here: https://github.com/zed-industries/test-tsconfig-issues/, and it seems like this isn't reproducing. If y'all could share more details on your setup or even better a repository that I could clone, I'd be happy to take a closer look :)
